README.md

```diff
@@ -1,71 +1,70 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
```

Comment:
(Optional) Consider editing the readme file to some required information about the project 
- Project Name and Description 
- Instructions to run the app